python/ray/tune/tune.py

```diff
@@ -300,7 +320,7 @@ def run(run_or_experiment,
         runner.step()
         if time.time() - last_debug > DEBUG_PRINT_INTERVAL:
             if verbose:
-                reporter.report(runner)
+                _report_progress(runner, progress_reporter)
```

Comment:
BTW, does it make sense to include the above "print_interval" casework inside the CLIReporter or not?

If not, maybe we should document the behavior somewhere?

Reply:
yeah that makes sense actually